NOTICE.md

```diff
@@ -23,6 +23,9 @@ Thanks goes to these wonderful people (sorted by last name):
  * Yoann Garcia
```

Comment:
We should remove the spaces at the beginning but also sort them by last name

Reply:
Done